### PR TITLE
Update banner with KubeCon event.json

### PR DESCRIPTION
**Pull request by anajankovic**
Description

Changes proposed in this pull request:

- KubeCon event to be added to the banner on the Kyma landing page

### CODE REVIEW

File: content/banner/slides.yml [hunk 1]
```diff
@@ -4,3 +4,11 @@ slides:
     url: "https://open.sap.com/courses/kyma1"
     startDate: "04/04/2019"
     endDate: "08/05/2019"
+    },
```
Comment (derberg) [inline content/banner/slides.yml]: please remove this line

File: content/banner/slides.yml [hunk 2]
```diff
@@ -4,3 +4,11 @@ slides:
...
+    {
```
Comment (derberg) [inline content/banner/slides.yml]: please remove this line

File: content/banner/slides.yml [hunk 3]
```diff
@@ -4,3 +4,11 @@ slides:
...
+  - text: "Join us for the KubeCon, Barcelona, May 21-23!"
+    startDate: "07/05/2019"
+    endDate: "23/05/2019"
+     }
```
Comment (derberg) [inline content/banner/slides.yml]: please remove this line, and next 3

**Comment by magicmatatjahu** [comment]
@anajankovic you will merge it?

**Pull request merged**